manageiq-operator/pkg/helpers/miq-components/postgresql.go

```diff
@@ -7,28 +7,32 @@ import (
 	resource "k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	intstr "k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
 func DefaultPostgresqlSecret(cr *miqv1alpha1.ManageIQ) *corev1.Secret {
 	labels := map[string]string{
 		"app": cr.Spec.AppName,
 	}
-	secret := map[string]string{
+
+	stringData := map[string]string{
 		"dbname":   "vmdb_production",
 		"username": "root",
 		"password": generatePassword(),
 		"hostname": "postgresql",
 		"port":     "5432",
 	}
 
-	return &corev1.Secret{
+	secret := &corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      postgresqlSecretName(cr),
 			Namespace: cr.ObjectMeta.Namespace,
 			Labels:    labels,
 		},
-		StringData: secret,
+		StringData: stringData,
 	}
+
+	return secret
```

Comment:
The logic here doesn't seem to have changed. Please only change what is necessary for the fix.

Also this was originally mentioned in https://github.com/ManageIQ/manageiq-pods/pull/441#discussion_r435219686